### PR TITLE
Update seurat-scripts to 4.4.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`seurat-scripts`](https://bioconda.github.io/recipes/seurat-scripts/README.html): **4.0.0** &rarr; **4.4.0**

[install with bioconda](http://bioconda.github.io/recipes/seurat-scripts/README.html) [Conda](https://anaconda.org/bioconda/seurat-scripts/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/seurat-scripts`](https://github.com//bioconda/bioconda-recipes/tree/bump/seurat_scripts/recipes/seurat-scripts) (click to view/edit other files)
Summary | A set of wrappers for individual components of the Seurat package.
Home | [https://github.com/ebi-gene-expression-group/r-seurat-scripts](https://github.com/ebi-gene-expression-group/r-seurat-scripts)
Releases |[https://github.com/ebi-gene-expression-group/r-seurat-scripts/tags](https://github.com/ebi-gene-expression-group/r-seurat-scripts/tags)
Recipe Maintainer(s) | @pcm32, @pinin4fjords, @nh3
Author | `@ebi-gene-expression-group`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by pcm32** [comment]
This requires some changes, please don't merge.

**Comment by pcm32** [comment]
@BiocondaBot please fetch artifacts

File: recipes/seurat-scripts/conda_build_config.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 r_base:
-  - 4.0
+  - 4.0.5
```
Comment (martin-g) [inline recipes/seurat-scripts/conda_build_config.yaml:2]: Does it have to be that specific version? Would it break with any newer version, 4.0.6 ?

Comment (pcm32) [reply recipes/seurat-scripts/conda_build_config.yaml:2]: This recipe has been a bit sensitive in the past, the format conversions parts have dependencies that easily break. The issue is probably not the r-base version, but the implications its version has on other packages (like the h5 related or loom).

**Review by mencian — APPROVED** [review]

**Pull request merged**